src/components/composer/composer/composer_store.ts

```diff
@@ -425,54 +298,33 @@ export class ComposerStore extends SpreadsheetStore {
     );
   }
 
-  private _stopEdition() {
+  protected _stopEdition() {
```

Comment:
```suggestion
  private _stopEdition() {
```

Reply:
It's used by both the cell_composer and standalone_composer...